src/aliceVision/depthMap/DepthSimMap.cpp

```diff
@@ -395,7 +395,8 @@ void DepthSimMap::saveToImage(const std::string& filename, float simThr) const
         metadata.push_back(oiio::ParamValue("AliceVision:storageDataType",
                                             EStorageDataType_enumToString(image::EStorageDataType::Float)));
         image::writeImage(filename, colorBuffer,
-                          image::OutputFileColorSpace(image::EImageColorSpace::NO_CONVERSION), metadata);
+                          image::ImageWriteOptions().toColorSpace(image::EImageColorSpace::LINEAR),
```

Comment:
Technical maps like depth, sim, etc have no notion of colorspace. In this case, we usually use NO_CONVERSION to make it explicit.

Reply:
Previously `OutputFileColorSpace` was setting both `from` and `to` to LINEAR if single-parameter constructor was used with NO_CONVERSION. This code preserves existing behavior.

See https://github.com/alicevision/AliceVision/pull/1282/commits/4668fe1999415a4ebbc8719559b406b568ccb039#diff-dc4ab367f585ae18a7872e28d2c057e6388eb99b2ff928a3a5295ffc0df14425L66